crates/subspace-service/src/dsn_archiver.rs

```diff
@@ -11,11 +12,14 @@ use tracing::{error, info, trace};
 pub fn start_subspace_dsn_archiver<Block>(
     subspace_link: &SubspaceLink<Block>,
     node_config: subspace_networking::Config,
-    spawner: &impl SpawnEssentialNamed,
+    task_manager: &TaskManager,
```

Comment:
Old spawner argument was fine actually, spawner is clonable if you need that (which I don't think is actually the case here).